frontend/morpheus/store/selectors.js

```diff
@@ -0,0 +1,19 @@
+export const selectCurrentRoom = state => state.currentRoom;
+
+export const selectRooms = state => state.rooms;
+
+export const selectCurrentUser = state => state.currentUser;
+
+export const selectOffice = state => state.office;
+
+export const selectOfficeFilter = state => state.officeFilter;
+
+export const selectUsers = state => state.users;
+
+export const selectUsersFilter = state => state.usersFilter;
+
+export const selectError = state => state.error;
+
+export const selectSettings = state => state.settings;
```

Comment:
I think this file generate an unnecessary dependency and complexity.
My suggestion is:
- use `mapStateToProps` only where necessary
- rely in default props with explicit required

Reply:
I don't understand, could you explain?